spaces/S000154/properties/P000061.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000154
+property: P000061
+value: false
+refs:
+- doi: 10.1007/978-1-4612-6290-9_6
+  name: Counterexamples in Topology
+---
+
+Suppose that $X$ is cozero complemented and let $U\subseteq \mathbb{R}$ be any countably infinite set. Then $U$ is an open $F_\sigma$-set and since {S154} is {S7} it follows that $U$ is a cozero set. Find a cozero set $V$ disjoint from $U$ such that $U\cup V$ dense in $X$. Since $X\setminus V$ is infinite we have $\infty\notin V$. Since $V$ is a cozero set, write $V = \bigcup_{n=1}^\infty F_n$ where $F_n$ are closed. Then $\infty\notin F_n$ so $F_n$ is finite. It follows that $V$ is countable. But then $X = \overline{U\cup V} = U\cup V\cup \{\infty\}$ is countable, contradiction.
```

Comment:
```suggestion
Suppose that $X$ is cozero complemented and let $U\subseteq \mathbb{R}$ be any countably infinite set. Then $U$ is an open $F_\sigma$-set and since {S154|P7} it follows that $U$ is a cozero set. Find a cozero set $V$ disjoint from $U$ such that $U\cup V$ is dense in $X$. Since $X\setminus V$ is infinite we have $\infty\notin V$. Since $V$ is a cozero set, write $V = \bigcup_{n=1}^\infty F_n$ where $F_n$ are closed. Then $\infty\notin F_n$ so $F_n$ is finite. It follows that $V$ is countable. But then $X = \overline{U\cup V} = U\cup V\cup \{\infty\}$ is countable, contradiction.
```